contracts-periphery/src/interfaces/ISwapRouter.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Nit, but I think we should call the folder `interface`, not `interfaces`—the singular version is what we/foundry use everywhere: `test` not `tests`, `script` not `scripts`, and `src` not `srcs`

Reply:
Renamed as suggested in https://github.com/ScopeLift/umbra-protocol/pull/341/commits/75faac7fcc1bbfed25b90ee04633145ddcd72693